woocommerce/rest-api/Debug_Controller.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * WooCommerce Plugin Framework
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the plugin to newer
+ * versions in the future. If you wish to customize the plugin for your
+ * needs please refer to http://www.skyverge.com
+ *
+ * @package   SkyVerge/WooCommerce/Plugin/Classes
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2013-2019, SkyVerge, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_5_0\REST_API;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_5_0\SV_WC_Payment_Gateway_Plugin;
+use SkyVerge\WooCommerce\PluginFramework\v5_5_0\SV_WC_Plugin;
+
+defined( 'ABSPATH' ) or exit;
+
+if ( ! class_exists( '\\SkyVerge\\WooCommerce\\PluginFramework\\v5_5_0\\REST_API\\Debug_Controller' ) ) :
+
+
+/**
+ * The plugin REST API Debug endpoint.
+ *
+ * @since 5.5.0-dev
+ */
+abstract class Debug_Controller extends \WC_REST_Controller {
+
+
+	/** @var SV_WC_Plugin main instance */
+	protected $plugin;
+
+	/** @var string endpoint namespace */
+	protected $namespace;
+
+	/** @var string the route base */
+	protected $rest_base;
+
+
+	/**
+	 * Debug controller constructor.
+	 *
+	 * @since 5.5.0-dev
+	 *
+	 * @param SV_WC_Plugin $plugin main instance
+	 */
+	public function __construct( SV_WC_Plugin $plugin ) {
+
+		$this->plugin    = $plugin;
+		$this->namespace = 'wc/v1';
+		$this->rest_base = $plugin->get_id();
```

Comment:
let's prefix this with `wc_` for consistency with how the plugin IDs are used throughout the framework

Reply:
but that's not the case in memberships though, this is not a PHP/framework context, rather API context - the endpoint would look like `wc/v3/wc_...` seems a bit weird to me